modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/StepsTable.scala

```diff
@@ -441,6 +445,7 @@ object StepsTable extends Columns {
   type ReceiveProps = ComponentWillReceiveProps[Props, State, Unit]
 
   private val MIDDLE_BUTTON = 1 // As defined by React.js
+  private val HEADER_ROW = -1
```

Comment:
I guess we want constants to follow the scala style `MiddleButton` and `HeaderRow`

Reply:
I had not fully internalized this convention for Scala constants. I have now ;) Renaming...